pkg/assembler/backends/neo4j/hasSLSA.go

```diff
@@ -114,7 +114,7 @@ func (c *neo4jClient) HasSlsa(ctx context.Context, hasSLSASpec *model.HasSLSASpe
 
 			sb.WriteString(query)
 			firstMatch = true
-			setPkgMatchValues(&sb, hasSLSASpec.Package, false, &firstMatch, queryValues)
+			setPkgMatchValues(&sb, hasSLSASpec.Subject.Package, false, &firstMatch, queryValues)
```

Comment:
same here will have to put a check in that `subject` is not nil for query all case

Reply:
Handled for now, but with e2e testing this might change in next PR(s)